src/ShopBundle/objects/db/TShopOrder.class.php

```diff
@@ -540,6 +543,32 @@ protected function SaveOrderNotificationDataToOrder($bOrderSend, $oMail, $sSendT
         $this->AllowEditByAll(false);
     }
 
+    protected function SetSendOrderNotificationState(bool $orderSend): void
```

Comment:
Naming: A "set" method should only do that (set things).

So the name should reflect what the method is doing.

Reply:
I changed the name accordingly